kb/2023-07-25/configure_priority_class_longhorn.md

```diff
@@ -0,0 +1,328 @@
+        if [ -z "$name" ]; then
+          break
+        fi
+        echo "Stop ${namespace}/${name}"
+        virtctl stop $name -n $namespace
+      done
+  ```
+
+  :::note
+    You can also stop all virtual machines from the Harvester GUI:
+    1. Navigate to the "Virtual Machines" page.
+    2. Select virtual machines and click the "Stop" button to stop them.
+  :::
+
+4. Ensure there are no running virtual machines:
+
+  Run the command:
+
+  ```bash
+  kubectl get vmi -A
+  ```
+
+  The above command must return:
+
+  ```bash
+  No resources found
+
+## Scale down monitoring pods
+
+1. Scale down the Prometheus deployment.
+
+  Run the command and wait for all Prometheus pods to terminate:
+
+  ```bash
+  kubectl patch -n cattle-monitoring-system prometheus/rancher-monitoring-prometheus --patch '{"spec": {"replicas": 0}}' --type merge && \
+      sleep 5 && \
+      kubectl rollout status --watch=true -n cattle-monitoring-system statefulset/prometheus-rancher-monitoring-prometheus
+  ```
+
+  A sample output looks like:
+
+  ```
+  prometheus.monitoring.coreos.com/rancher-monitoring-prometheus patched
+  statefulset rolling update complete 0 pods at revision prometheus-rancher-monitoring-prometheus-cbf6bd5f7...
+  ```
+
+2. Scale down the AlertManager deployment.
+
+  Run the command and wait for all AlertManager pods to terminate:
+
+  ```bash
+  kubectl patch -n cattle-monitoring-system alertmanager/rancher-monitoring-alertmanager --patch '{"spec": {"replicas": 0}}' --type merge && \
+      sleep 5 && \
+      kubectl rollout status --watch=true -n cattle-monitoring-system statefulset/alertmanager-rancher-monitoring-alertmanager
+  ```
+
+  A sample output looks like this:
+
+  ```
+  alertmanager.monitoring.coreos.com/rancher-monitoring-alertmanager patched
+  statefulset rolling update complete 0 pods at revision alertmanager-rancher-monitoring-alertmanager-c8c459dff...
+  ```
+
+3. Scale down the Grafana deployment.
+
+  Run the command and wait for all Grafana pods to terminate:
+
+  ```bash
+  kubectl scale --replicas=0 deployment/rancher-monitoring-grafana -n cattle-monitoring-system && \
+      sleep 5 && \
+      kubectl rollout status --watch=true -n cattle-monitoring-system deployment/rancher-monitoring-grafana
+  ```
+
+  A sample output looks like this:
+
+  ```
+  deployment.apps/rancher-monitoring-grafana scaled
+  deployment "rancher-monitoring-grafana" successfully rolled out
+  ```
+
+## Scale down vm-import-controller pods
+
+Check if the [`vm-import-controller` addon](https://docs.harvesterhci.io/v1.1/advanced/vmimport) is enabled and configured with a persistent volume:
+
+```bash
+kubectl get pvc -n harvester-system harvester-vm-import-controller
+```
+
+If the above command returns an output like this, you must scale down the `vm-import-controller` pod. Otherwise, you can skip the step.
```

Comment:
```suggestion
If the above command returns an output like this, you must scale down the `vm-import-controller` pod. Otherwise, you can skip the following step.
```